careers/devops.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: DevOps Engineer
+summary: The Haskell Foundation seeks a passionate DevOps professional to support GHC, related Haskell projects and tooling, and the HF itself.
+postedOn: 2021-12-28
+status: Open
+---
+
+# Job Details
+
+## Responsibilities
+
+* Maintain GitLab and CI infrastructure for GHC
+* Improve CI
+* Expand the CI infrastructure to include base the base Haskell ecosystem
+* Support other Haskell libraries and tools, including HLS (Haskell Language Server)
+* Work with the Haskell Infrastructure Team
```

Comment:
This is vague to me. Would you expect the hire to, essentially, become the functional arm of the infra team? Help out when needed? Somewhere in between? It might be good to get an idea from the infra team what they would hope from this position.

Also good would be defining the infra team, with a link to https://github.com/haskell-infra/haskell-admins

Reply:
we don't imagine there will be much at first -- just over time we'd ideally integrate this person into the infra team as time becomes available and projects arise.